Test

onejmi opened the pull request:
merge into master

Commit:
initial test commit

Commit:
Created a Project Notes text file

Commit:
Updated notes and assigned sections of work

Commit:
gyro setup #1

onejmi committed:
added turnvalue log

Commit:
Added class DriveStraightTimed
Rename DriveStraight to DriveStraightDistance

Commit:
Added MPU6050 Gyro/Accelerometer utility class

rakusan2 reviewed (changes requested):
binary's need to be deleted
.gitignore needs to be returned

Commit:
Added Team3571.XboxController version to test branch. 
Create RobotMap and updated other classes to reflect this change.
Integrated XboxController into test branch code

Commit:
Adding to test

Commit:
Added to test branch

Commit:
Added Encoder Values

Commit:
Updated RobotMap constants

Commit:
Added class ShiftGears command, updated RobotMap and Pneumatics

Commit:
Updated RobotMap

Commit:
Adding ShiftGears command class, Updated RobotMap

Commit:
Added ShiftGears and .gitignore

Commit:
Updated Pneumatics

Commit:
Added Intake subsystem and BoxIntake command. Updated RobotMap and Robot
